rules/opentitan.bzl

```diff
@@ -318,10 +555,18 @@ def verilator_params(
         @param tags: The test tags to apply to the test rule.
         @param timeout: The timeout to apply to the test rule.
         @param local: Whether the test should be run locally and without sandboxing.
-        @param args: Arguments to pass to the test.
+        @param args: Extra arguments to pass to `opentitantool`.
```

Comment:
args is problematically non-descript and it looks like you're trying to solve it with a comment. It's universally non-descript so I don't think you're under any obligation to be the one to fix it.

Reply:
Yeah, I noticed that, and perhaps it should be renamed to "opentitantool_args" or something like that? I didn't want to cram changes into this PR that were not specifically related to the goal of generating two types of artifacts (one for flash and one for ROM). Happy to address this in a follow up PR though if you would like. Just lmk!